### ISSUE TITLE
lower/upper case in scientific output format

### ISSUE THREAD

**Issue by slayoo** [gnudatalanguage/gdl#93]
As reported on SF.net (although with different symptoms): https://sourceforge.net/p/gnudatalanguage/bugs/343/

```
IDL> print, 1.2, format='(%"%e")'
1.200000e+00
IDL> print, 1.2, format='(%"%E")'
1.200000E+00
IDL> print, 1.2, format='(E)'
1.2000000E+00
IDL> print, 1.2, format='(e)'
1.2000000e+00
```

while

```
GDL> print, 1.2, format='(%"%e")'
1.200000E+00
GDL> print, 1.2, format='(%"%E")'
1.200000E+00
GDL> print, 1.2, format='(E)'
  1.2000000E+00
GDL> print, 1.2, format='(e)'
  1.2000000E+00
```

**Comment by GillesDuvert** [comment]
solved by #197

**Comment by GillesDuvert** [comment]
reopened as to close it automatically with proper pull request
